src/main/java/com/rescuebites/api/client/controllers/requests/CreateClientRequest.java

```diff
@@ -26,5 +26,5 @@ public record CreateClientRequest(
         @NotNull(message = "El usuario es obligatorio")
         UUID userId,
 
-        List<UUID> preferenceIds
+        List<PreferenceType> preferences
 ) {}
```

Comment:
Siguiendo el comentario anterior, vos podés setear valores por defectos de los atributos si esto en vez de ser un record fuese una clase común y corriente. Algo así:


```suggestion
public class CreateClientRequest {
        @NotBlank(message = "El nombre es obligatorio")
        String firstName,
        
        @NotBlank(message = "El apellido es obligatorio")
        String lastName,
        
        @PastOrPresent(message = "La fecha de nacimiento no puede ser futura")
        @NotNull(message = "La fecha de nacimiento es obligatoria")
        LocalDate birthDate,
        
        @NotBlank(message = "La dirección es obligatoria")
        String address,
        
        @NotNull(message = "El usuario es obligatorio")
        UUID userId,

        List<PreferenceType> preferences = Collections.emptyList();
}
```